var/spack/repos/builtin/packages/gtkplus/package.py

```diff
@@ -30,7 +30,9 @@ class Gtkplus(AutotoolsPackage):
        interfaces for applications."""
     homepage = "http://www.gtk.org"
     url = "http://ftp.gnome.org/pub/gnome/sources/gtk+/2.24/gtk+-2.24.31.tar.xz"
-
+    version('3.20.10',
+            'e81da1af1c5c1fee87ba439770e17272fa5c06e64572939814da406859e56b70',
+            url='http://ftp.gnome.org/pub/gnome/sources/gtk+/3.20/gtk+-3.20.10.tar.xz')
```

Comment:
Can you override `url_for_version` so that this version-specific URL becomes unnecessary?